docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.install

```diff
@@ -1026,3 +1028,113 @@ function su_humsci_profile_update_9714() {
     $role->save();
   }
 }
+
+/**
+ * Remove any existing hs_webform paragraph entities.
+ */
+function su_humsci_profile_update_9715($sandbox) {
+  $paragraph_storage = \Drupal::entityTypeManager()->getStorage('paragraph');
+  $paragraph_machine_name = 'hs_webform';
+
+  if (empty($sandbox['ids'])) {
```

Comment:
I doubt there's more than a dozen on any given site, but it's good practice to use the sandbox when making entity changes that might be of an unknown quantity. :+1: